docs/tutorials/private-registries.md

```diff
@@ -0,0 +1,162 @@
+# Allow the Trivy Operator to access private registries
+
+In this tutorial, we will detail multiple ways on setting up the Trivy Operator to access and scan container images from private container registries.
+
+## Prerequisites
+
+To follow this tutorial, you need the following installed on your machine:
+
+- The Helm CLI tool
+- kubectl and connected to a running Kubernetes cluster
+- a container images in a private registy, running as a pod inside your cluster
+
+Note that we will be using a local Kubernetes KinD cluster and a private container image stored on a private GitHub repository for the examples.
+
+## First Option: Filesystem Scanning
+
+For this tutorial, we will use the [Operator Helm Chart.](https://aquasecurity.github.io/trivy-operator/latest/operator/installation/helm/)
+
+The configuration options for the Helm Chart can be found in the [values.yaml](https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/values.yaml) manifest.
+Navigate to the section `Trivy.command`. The default will be:
+
+```
+trivy:
+# command. Either `image` or `filesystem` scanning, depending on the target type required for the scan.
+  # For 'filesystem' scanning, ensure that the `trivyOperator.scanJobPodTemplateContainerSecurityContext` is configured
+  # to run as the root user (runAsUser = 0).
+  command: image
+```
+
+By default, the command that trivy is supposed to run inside your cluster is `trivy image` for container image scanning. However, we want to change it to scan the filesystem in your nodes instead. Container images are ultimately stored as files on the node level of your cluster. This way, trivy is going to scan the files of your container images for vulnerabilities. This is a little bit of a work-around with the downside that the Trivy Operator will have to run as root. However, remember that security scanning already requires the operator to have lots of cluster privileges.
+
+Next, we will change the the `command` and the `trivyOperator.scanJobPodTemplateContainerSecurityContext`of the values.yaml manifest. For this, we can create a new values.yaml manifest with our desired modifiactions:
+```
+trivy:
+    command: fs
+    ignoreUnfixed: true
+trivyOperator:
+    scanJobPodTemplateContainerSecurityContext:
+        # For filesystem scanning, Trivy needs to run as the root user
+        runAsUser: 0
+```
+
+Lastly, we can deploy the operator inside our cluster with referencing our new values.yaml manifest to override the default values:
+
+```
+helm upgrade --install trivy-operator aqua/trivy-operator \
+  --namespace trivy-system \
+  --create-namespace \
+  --version 0.1.9
+  --values ./values.yaml
```

Comment:
also can be achieved in this way
```suggestion
helm upgrade --install trivy-operator aqua/trivy-operator \
  --namespace trivy-system \
  --create-namespace \
  --version 0.1.9
  --set="trivy.command=fs"
```

Reply:
@chen-keinan but you would also have to give the operator root access at the same time, no?

```
helm upgrade --install trivy-operator aqua/trivy-operator \
  --namespace trivy-system \
  --create-namespace \
  --version 0.1.9
  --set="trivy.command=fs"
  --set="trivyOperator.scanJobPodTemplateContainerSecurityContext.runAsUser=0"
```